torchreid/models/mobilenetv3.py

```diff
@@ -240,7 +263,36 @@ def _initialize_weights(self):
                 m.weight.data.normal_(0, 0.01)
                 m.bias.data.zero_()
 
-def init_pretrained_weights(model, key=''):
+
+class MobileNetV3LargeTimm(MobileNetV3Base):
+    def __init__(self,
+                pretrained=False,
+                **kwargs):
+
+        super().__init__(**kwargs)
+        self.model = timm.create_model('mobilenetv3_large_100_miil_in21k', pretrained=pretrained, num_classes=self.num_classes)
+        self.use_angle_simple_linear = False
+        self.dropout = Dropout(**self.dropout_cls)
+        self.loss = 'softmax'
```

Comment:
What if `am_softmax` was initially passed to constructor? I think an error should be thrown in that case.